src/config/SuffixList.json

```diff
@@ -1,86 +1,111 @@
 [
   {
```

Comment:
Great. This is much better as it will allow the flexibility to add more workflows.